src/Error/ExceptionTrap.php

```diff
@@ -235,6 +233,19 @@ public function handleException(Throwable $exception): void
         }
     }
 
+    /**
+     * Apply callbacks to an exception.
+     *
+     * @param \Throwable $exception Exception instance.
+     * @return void
+     */
+    public function applyCallbacks(Throwable $exception): void
```

Comment:
Is this public just for testing?

Reply:
Its public because I needed a way to call this method from the middleware.

Reply:
This doesn't feel great to me. Maybe it's the name. Should we be relying on something less janky like events to run through the error handling steps?

Reply:
Yeah, events would be better. I'll open another pull request to remove callbacks and use events instead.